### PR TITLE
[12.0] porting stock_analysis

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: stock_analysis/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# © 2020 Lorenzo Battistini - TAKOBI
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Analysis",
+    "summary": "Analysis view for stock",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Inventory, Logistic, Storage",
```
Comment (rousseldenis) [inline stock_analysis/__manifest__.py]: Change this to 'Warehouse' as it won't appear in good category on OCA shop.

File: stock_analysis/report/stock_analysis.py [hunk 2]
```diff
@@ -10,12 +9,13 @@ class StockAnalysis(models.Model):
...
     _auto = False
     _rec_name = 'product_id'
+    _description = 'Stock analysis view'
 
     product_id = fields.Many2one(
         'product.product', string='Product', readonly=True)
     location_id = fields.Many2one(
         'stock.location', string='Location', readonly=True)
-    qty = fields.Float(string='Quantity', readonly=True)
+    quantity = fields.Float(string='Quantity', readonly=True)
```
Comment (rousseldenis) [inline stock_analysis/report/stock_analysis.py:18]: Even in a view, I think you should add decimal precision

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. At least manifest change.

File: stock_analysis/__manifest__.py (hunk 1 above)
Comment (eLBati) [reply stock_analysis/__manifest__.py]: Thanks, done

File: stock_analysis/report/stock_analysis.py (hunk 2 above)
Comment (eLBati) [reply stock_analysis/report/stock_analysis.py:18]: This is based on `quantity` fields of `stock.quant`, that does not have precision

**Comment by eLBati** [comment]
@rousseldenis @gurneyalex do you know why there is not runbot here?
Thanks

**Comment by rousseldenis** [comment]
@gurneyalex Indeed, runbot builds do not seem to be launched here.

**Review by mrcast — APPROVED** [review]
I tested the functionality on local database
Functional test ok

**Comment by eLBati** [comment]
@rousseldenis could you update your review?

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**